OSDOCS#13367:Replace/Insight Operator/Insights Operator (#91797)
openshift/openshift-docs: modules/obfuscating-dvo-data.adoc

[id="obfuscating-deployment-validation-operator-data_{context}"]
= Obfuscating Deployment Validation Operator data

> Cluster administrators can configure the Insights Operator to obfuscate data from the Deployment Validation Operator (DVO), if the Operator is installed. When the `workload_names` value is added to the *insights-config* `ConfigMap` object, workload names—rather than UIDs—are displayed in Insights for Openshift, making them more recognizable for cluster administrators. 

Reviewer: IMHO, a comma before `if` is not needed.
Also, a rewording suggestion to consider:
```suggestion
As a cluster administrator, you can configure the Insights Operator to obfuscate data from the optionally installed Deployment Validation Operator (DVO).

To display recognizable workload names rather than UIDs in *Insights for Openshift*, you can add the `workload_names` parameter to the *insights-config* `ConfigMap` object.
```
